switch-configuration/config/vlans.d/Hilton

```diff
@@ -0,0 +1,18 @@
+//	Conference	Center	--	VLANS	100-499
+VLAN	hiSCALE-SLOW		100	2001:470:f0fb:100::/64	10.128.128.0/21	2.4G Wireless Network in 
+VLAN	hiSCALE-FAST		101	2001:470:f0fb:101::/64	10.128.136.0/21	5G Wireless Network in 
+VLAN	hiSpeaker		102	2001:470:f0fb:102::/64	10.128.2.0/24	Speaker Network
+VLAN	hiInfra			103	2001:470:f0fb:103::/64	10.128.3.0/24	SCALE Network Infrastructure and Servers
+VLAN	hiCTF			104	2001:470:f0fb:104::/64	10.128.4.0/24	Capture the Flag
+VLAN	hiAVLAN			105	2001:470:f0fb:105::/64	10.128.5.0/24	Audio Visual Network (DHCP Helper to AV server)
+VLAN	hiNOC			106	2001:470:f0fb:106::/64	10.128.6.0/24	Wired network in NOC
+VLAN	hiSigns			107	2001:470:f0fb:107::/64	0.0.0.0/0	Signs network (IPv6 Only)
+VLAN	hiStaff			108	2001:470:f0fb:108::/64	10.128.8.0/24	Staff Wifi Network
+VLAN	hiInstall		110	2001:470:f0fb:110::/64	10.128.10.0/24	Install Fests and Workshops
```

Comment:
is there a reason we've swapped out the vlan 110? Orginally it was reg and now it looks like its installs (install fests, etc.). Do we need to factor in reg this time around or what are your thoughts there?

ExReg (vlans.d/Expo)
```
VLAN    exRegistration          110     2001:470:f325:110::/64  10.0.10.0/24    Registration Network (Expo Center) IPv6 Only
```

HiInstall (vlans.d/Hilton)
```
VLAN    hiInstall               110     2001:470:f0fb:110::/64  10.128.10.0/24  Install Fests and Workshops
```

Reply:
Because even though we're using EX VLAN numbers and EX network addresses, most of the Hilton is actually templated from CF files which were closer to what we needed. Registration is unique to EX and so it got added back in. Likely Install was previously 410.